### PR TITLE
Fixes incorrect contribution guideline

### LINKED ISSUE #186: Probe Engine Environment Variable Updation
Description
When running the application locally the application is failing due to the probe engine environment variable of being a 7 character .

### PR DESCRIPTION
**Pull request by alecharp**
Resolves #186.

cc @impact-maker

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -8,5 +8,7 @@ GITHUB_APP_ID=token_example
 GITHUB_APP_PRIVATE_KEY_PATH=path-to-private-key
 GITHUB_APP_INSTALLATION_NAME=organization-or-username
 
-UPDATE_CENTER_CRON=* 0 */2 * * *
-PROBE_ENGINE_CRON=* 5 */2 * * * *
+UPDATE_CENTER_CRON=0 0 */2 * * *
+# This means on the hour, every 2 hours
+PROBE_ENGINE_CRON=0 5 */2 * * *
+# This means at 5, every 2 hours
```
Comment (lemeurherve) [inline .env.example:14]:
```suggestion
# This means on the hour, every 2 hours
UPDATE_CENTER_CRON=0 0 */2 * * *
# This means at 5, every 2 hours
PROBE_ENGINE_CRON=0 5 */2 * * *
```

**Comment by dheerajodha** [comment]
Thanks for the PR. This would solve the issue that new contributors are seeing.

Maybe a question for another PR: I was wondering if we want to also include some additional instructions in the CONTRIBUTING.adoc? For example:

`$ npm install` then
`$ yarn install` then
`$ yarn dev`.

**Comment by dheerajodha** [comment]
And also include a section in the doc that can help contributors access the DB and peek into the tables. For example:

`$ psql $POSTGRES_DB -h $POSTGRES_HOST -U $POSTGRES_USER`
`# select * from plugins;`
`# select * from scores;`

**Comment by alecharp** [comment]
> Thanks for the PR. This would solve the issue that new contributors are seeing.
> 
> Maybe a question for another PR: I was wondering if we want to also include some additional instructions in the CONTRIBUTING.adoc? For example:
> 
> `$ npm install` then `$ yarn install` then `$ yarn dev`.

you should never do `npm install`. That's why it's not in the documentation.

**Review by dheerajodha — APPROVED** [review]

**Comment by dheerajodha** [comment]
What about adding a line about installing the GitHub App into your account?

File: CONTRIBUTING.adoc [hunk 2]
```diff
@@ -53,6 +56,12 @@ openssl pkcs8 -topk8 -inform PEM -outform PEM -in ORIGINAL_KEY.pem -out CONVERTE
 
 NOTE: You need to change `ORIGINAL_KEY` and `CONVERTED_KEY` with real values.
 
+This final file is the one to point to in `GITHUB_APP_PRIVATE_KEY_PATH` environment variable.
+
+Finally, you need to install the GitHub APP on your GitHub user.
+In the GitHub App configuration page, find the *Install App* like on the left side menu.
```
Comment (dheerajodha) [inline CONTRIBUTING.adoc:62]: Optional
```suggestion
In the GitHub App configuration page, find the *Install App* option on the left side menu.
```

Comment (alecharp) [reply CONTRIBUTING.adoc:62]: it was link, not like. thanks for finding that typo.

**Pull request merged**